msg/CartesianTrajectory.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+
+# The name of the Cartesian frame the trajectory describes it's motion of.
+string frame
```

Comment:
``Header`` already has a ``frame_id``, so we shouldn't need ``frame``.

Reply:
The header carries the reference frame id, this is the target. I think in the first version I called it `end effector frame` but decided to shorten it.